lib/ansible/modules/system/pamd.py

```diff
@@ -186,16 +189,37 @@
 """
 
 RETURN = '''
-dest:
-    description: path to pam.d service that was changed
+change_count:
+    description: How many rules were changed
+    type: int
+    sample: 1
     returned: success
+new_rule:
+    description: The changes to the rule
     type: string
-    sample: "/etc/pam.d/system-auth"
+    sample: None      None None sha512 shadow try_first_pass use_authtok
+    returned: success
+updated_rule_(n):
+    description: The rule(s) that was/were changed
+    type: string
+    sample:
+    - password      sufficient  pam_unix.so sha512 shadow try_first_pass
+      use_authtok
+    returned: success
+result:
+    description:
+    - "That action that was taken and is one of: update_rule,
+      insert_before_rule, insert_after_rule, args_present, args_absent,
+      absent."
+    returned: always
+    type: string
+    sample: "update_rule"
```

Comment:
you should note that the return changed and those fields are new in 2.4 .. also keep dest and note that it is only valid in previous versions